crates/sui-source-validation-service/src/lib.rs

```diff
@@ -174,7 +174,8 @@ pub async fn verify_package(
             /* verify_deps */ false,
             SourceMode::Verify,
         )
-        .await?;
+        .await
+        .map_err(|e| anyhow!("Network {}: {}", network, e))?;
```

Comment:
When on-chain verification fails for some package, log which network is out of sync